pod/perlfunc.pod

```diff
@@ -4348,7 +4348,8 @@ X<my>
 A L<C<my>|/my VARLIST> declares the listed variables to be local
 (lexically) to the enclosing block, file, or L<C<eval>|/eval EXPR>.  If
 more than one variable is listed, the list must be placed in
-parentheses.
+parentheses. Variables should not subsequently be redeclared in the
```

Comment:
The word *subsequently* there seems superfluous.  I think removing it would remove a whiff of ambiguity from that sentence.

Reply:
I subsequently went with Grinnz' wording below, with a minor tweak.